yarn.lock

```diff
@@ -2,97 +2,108 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
-  version "7.0.0"
-  resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
+"@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35", "@babel/code-frame@^7.5.5":
```

Comment:
Hate to suggest this because we don't really understand why the `yarn.lock` has all of these changes, but since we didn't add any new dependencies, can we restore it to master? I think `git checkout master yarn.lock` would do i t

Reply:
Done! 👍